.openapi-generator/FILES

```diff
@@ -71,6 +71,7 @@ lib/winthrop-client-ruby/api/scraper_api.rb
 lib/winthrop-client-ruby/api_client.rb
 lib/winthrop-client-ruby/api_error.rb
 lib/winthrop-client-ruby/configuration.rb
+lib/winthrop-client-ruby/refresh_token.rb
```

Comment:
This is an intermediate file you don't need to edit, though this line will get removed on the next rake run anyway.